src/parser/internal.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
 import * as md from '../markdown';
```

Comment:
I think we'd like to keep prettier enabled and maybe revisit the config if the settings aren't readable